### ISSUE TITLE
UDP Panic on Windows

### ISSUE THREAD

**Issue by tsynik** [anacrolix/torrent#712]
I have reports of panic with UDP on Windows. Disabling uTP for torrent doesn't help. Here is panic (sorry for image)
![UDP-Err](https://user-images.githubusercontent.com/1551446/149143311-08adbb19-8f61-4a83-a74f-32d551da86e3.png)
:

**Comment by tsynik** [comment]
Hm, found his commit https://github.com/anacrolix/torrent/commit/aaec4dbd80100702f8a230b0a21309726caeaadb#diff-c50a2bc1fe6acbf03d37a550b2daa8d04c384e42cf9072a8d392fd9cffb35558 so this issue is known. Do I need to close this?

**Comment by anacrolix** [comment]
No, keep this open. This looks similar to https://github.com/anacrolix/torrent/issues/83. Perhaps it's worth investigating if the underlying libraries have advanced sufficiently to ignore or handle this error more appropriately now?
